portfolio/src/main/webapp/index.html

```diff
@@ -1,175 +1,245 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <link href='https://fonts.googleapis.com/css?family=Aclonica' rel='stylesheet'>
+    <meta charset="UTF-8" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Aclonica"
+      rel="stylesheet"
+    />
     <title>[Parth Vora] - Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
   </head>
 
   <body>
     <div id="content">
-        <div class="header">
-            <a href="#default" class="logo">Parth Vora</a>
-            <div class="header-right">
+      <div class="header">
+        <a href="#default" class="logo">Parth Vora</a>
+        <div class="header-right"></div>
```

Comment:
FYI, blank elements like this can be writen more succinctly:
```suggestion
        <div class="header-right" />
```